.github/workflows/velox_backend.yml

```diff
@@ -68,12 +70,24 @@ jobs:
           path: |
             ./cpp/build/releases/
           key: cache-velox-build-centos-7-${{ hashFiles('./cache-key') }}
+      - name: Get Ccache
+        uses: actions/cache/restore@v3
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-centos7-release-default
       - name: Build Gluten native libraries
         if: ${{ steps.cache.outputs.cache-hit != 'true' }}
         run: |
           df -a
+          yum install ccache -y
           cd $GITHUB_WORKSPACE/
           bash dev/ci-velox-buildstatic-centos-7.sh
+      - name: "Save ccache"
+        uses: actions/cache/save@v3
+        id: ccache
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-centos7-release-default
```

Comment:
@zhouyuan, do we need to save ccache here? If so, I guess every build triggered for different pr will save and overwrite existing ccache. Maybe, only save ccache in velox_backend_cache.yml on push commit event?

Reply:
> @zhouyuan, do we need to save ccache here? If so, I guess every build triggered for different pr will save and overwrite existing ccache. Maybe, only save ccache in velox_backend_cache.yml on push commit event?

Per GitHub's design PR doesn't access caches created by other PRs. Perhaps storing caches in PR might accelerate subsequent build in the same PR, we may need to test. But I don't feel strongly of doing this too since it might affect build stability among commits in the same PR.